deep_beamline_simulation/ophyd_attempt.py

```diff
@@ -21,7 +21,7 @@
 print()
 print(response_auth_guest_login.url)
 pprint.pprint(response_auth_guest_login.json())
-
+cook = response_auth_guest_login.cookies
```

Comment:
always using `session.post(...)` will take care of this and also has some performance benefits